### PR TITLE
Add small changes to feature-visualization

### PR DESCRIPTION
**Pull request by monicadragan**
Add the following changes: 
(1) min hight for lollipops, 
(2) fixed num of decimals for frequency in the table, 
(3) remove duplicated function createSummaryTable, 
(4) update terms for gene annotations (this was done in commit b776893b65363cd0d8392b2cf10b3fd639ca949f, but I think you didn't sync when cloning the branch)

### CODE REVIEW

**Comment by kpj** [comment]
Thanks!

What is the purpose of the min. lollipop height (only aesthetics)? Is the chosen threshold small enough?

**Comment by monicadragan** [comment]
> Thanks!
> 
> What is the purpose of the min. lollipop height (only aesthetics)? Is the chosen threshold small enough?

Yes, I would set a min height such that you ca see a bit of the tail of the lollipop. I lowered the min height now. I can remove this if you think it's no necessary.

**Comment by kpj** [comment]
> Yes, I would set a min height such that you ca see a bit of the tail of the lollipop. I lowered the min height now. I can remove this if you think it's no necessary.

Maybe we can leave it out for, in order to reduce confusions about low frequency lollipops.
I don't know if this is a real problem though.

**Comment by monicadragan** [comment]
> > Yes, I would set a min height such that you ca see a bit of the tail of the lollipop. I lowered the min height now. I can remove this if you think it's no necessary.
> 
> Maybe we can leave it out for, in order to reduce confusions about low frequency lollipops.
> I don't know if this is a real problem though.

ok, I removed the min height. 
Can I merge now?

File: scripts/assemble_visualization_webpage.py [hunk 1]
```diff
@@ -25,9 +25,9 @@ def convert_vcf(fname):
                     "position": record.POS,
                     "reference": record.REF,
                     "variant": [v.sequence for v in record.ALT],
-                    "frequency": np.mean(
+                    "frequency": round(np.mean(
                         [v for k, v in record.INFO.items() if k.startswith("Freq")]
-                    ),
+                    ),3),
```
Comment (kpj) [inline scripts/assemble_visualization_webpage.py]: There should be a space between `,` and `3` 😄

**Review by kpj — APPROVED** [review]

File: scripts/assemble_visualization_webpage.py (hunk 1 above)
Comment (monicadragan) [reply scripts/assemble_visualization_webpage.py]: Done.

**Pull request merged**